new-pipeline/src/dist.js

```diff
@@ -1850,8 +1850,8 @@ function saveStateToUrlAndCookie() {
   }
 
   var isKeyedHistogram = $("#sort-keys-option").css("display") !== "none";
-  if (gInitialPageState.use_submission_date !== 0 || gInitialPageState.table !==
-    0 || gInitialPageState.cumulative !== 0 || gInitialPageState.trim !== 1 ||
+  if (gInitialPageState.use_submission_date == 0 || gInitialPageState.table ==
```

Comment:
The current defaults should be left alone in order to not surprise users who are used to the default behaviour.